### PR TITLE
BOOKSTORE

### PR DESCRIPTION
**Pull request by AACHOURMOHAMED**
# BOOKSTORE

>Add books component
>Add book component
>Add router

### CODE REVIEW

File: src/components/Layouts/Book.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Input from '../UI/Input';
+import classes from './Book.module.scss';
+import Card from '../UI/Card';
+
+const Book = (props) => {
+  const { book, author } = props;
```
Comment (EricMbouwe) [inline src/components/Layouts/Book.js]: - [x] Takes the book data as a prop and renders the `title `and the `author`. Let's replace **book** by **title** in the props

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @AACHOURMOHAMED, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Linters ✔
- PR ✔
- Components structure ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Layouts/Book.js (hunk 1 above)
Comment (AACHOURMOHAMED) [reply src/components/Layouts/Book.js]: done

**Review by cynthiainga — APPROVED** [review]
Hi @AACHOURMOHAMED, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**